app/subscriber/src/features/my-folders/ConfigureFolder.tsx

```diff
@@ -49,7 +62,16 @@ export const ConfigureFolder: React.FC = () => {
   }, []);
 
   React.useEffect(() => {
-    if (!currentFolder && id) {
+    currentFolder &&
+      !currentFolder.events.length &&
+      setCurrentFolder({
+        ...currentFolder,
+        events: [createSchedule(currentFolder.name, currentFolder.description)],
+      });
+  }, [currentFolder]);
+
+  React.useEffect(() => {
+    if ((!currentFolder && id) || currentFolder?.id !== Number(id)) {
```

Comment:
when user changes selected item on left side, or when folder is not set

Reply:
If the folder doesn't exist, won't this infinite loop?